integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -222,6 +227,16 @@ public static void startIntegrationTest() throws Exception {
     } else {
       client = GravitinoAdminClient.builder(serverUri).build();
     }
+
+    Awaitility.await().atMost(60, TimeUnit.SECONDS).pollInterval(5, TimeUnit.SECONDS).until(() -> {
```

Comment:
5 seconds seems too long, can you shorten it to 1 second or so?

Reply:
ok! I will modify it.